stays/forms.py

```diff
@@ -0,0 +1,35 @@
+from .models import Stay, Review
+from django.forms import ModelForm
+from django.forms import ValidationError
+
+
+class StayForm(ModelForm):
+    class Meta:
+        model = Stay
+        fields = ['owner', 'provider', 'start_date', 'end_date', 'pets']
+
+    def clean(self):
+        cleaned_data = super().clean()
+        owner = cleaned_data.get("owner")
+        provider = cleaned_data.get("provider")
+        start_date = cleaned_data.get("start_date")
+        end_date = cleaned_data.get("end_date")
+        if owner == provider.person:
+            self.add_error('provider', 'You cannot perform a stay for yourself')
+        if end_date < start_date:
```

Comment:
You should also create a `clean_start_date` and `clean_end_date` such that they are not less than the current date